src/pages/organisationUnits/form/ImageField.tsx

```diff
@@ -87,7 +87,13 @@ export function ImageField() {
     }
 
     return (
-        <UIField label={i18n.t('Image')} name="image">
+        <UIField
+            label={i18n.t('Image')}
+            name="image"
+            error={!!(input.value && input.value.error)}
+            valid={!!(input.value && input.value.id)}
+            validationText={input.value?.error}
```

Comment:
Generally we use `meta` for validation state. 

eg. 
```suggestion
            error={meta.invalid}
            valid={meta.valid}
            // only show error if touched as well
            validationText={meta.touched && meta.error?.toString()}
```